code/modules/mob/living/living.dm

```diff
@@ -277,17 +277,19 @@
 
 /mob/living/resist_grab(moving_resist)
 	if(pulledby.grab_level)
-		if(prob(30/pulledby.grab_level))
+		grab_resist_level += 1
+		if(grab_resist_level > pulledby.grab_level)
 			playsound(src.loc, 'sound/weapons/thudswoosh.ogg', 25, 1, 7)
 			visible_message("<span class='danger'>[src] has broken free of [pulledby]'s grip!</span>", null, null, 5)
 			pulledby.stop_pulling()
-			return 1
+			grab_resist_level = 0 //zero it out.
+			return TRUE
 		if(moving_resist && client) //we resisted by trying to move
```

Comment:
Use the proper changeNextMove proc now that the click rework is merged here

Reply:
I don't know what you're talking about.

Reply:
clinet.next_movement shouldn't be used, instead do something like
`usr.changeNext_move(value)`